modules/rhcos-load-firmware-blobs.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="rhcos-load-firmware-blobs_{context}"]
+
+= Loading custom firmware blobs in the machine config manifest
+
+Because the default location for firmware blobs in `/usr/lib` is read-only, you can locate a custom firmware blob by updating the search path. This enables you to load local firmware blobs in the machine config manifest when the blobs are not managed by {op-system}.
+
+.Procedure
+
+. Create a Butane config file, `98-worker-firmware-blob.bu`, that updates the search path so that it is root-owned and writable to local storage. The following example places the custom blob file under `/var/lib/firmware`.
++
+[NOTE]
+====
+See "Creating machine configs with Butane" for information about Butane.
+====
+.Butane config file for custom firmware blob
++
+[source,yaml]
+----
+variant: openshift
+version: 4.9.0
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 98-worker-firmware-blob
+storage:
+  files:
+  - path: /FWPATH/<package_name> <1>
+    mode: 0644 <2>
+spec:
+  kernelArguments:
+    - 'firmware_class.path=/var/lib/firmware' <3>
```

Comment:
It should be noted that the value provided to `firmware_class.path` should be used for `$FWPATH` in the rest of the Butane example

Reply:
Should this be reflected in line 31 of the example then (storage.files.path)? I've updated that line to change `$FWPATH/<package_name>` to `/var/lib/firmware/<package_name>'.
If that's not the case, feel free to suggest an inline edit or we can discuss for clarification.